_includes/configuration-advanced.md

```diff
@@ -26,66 +26,62 @@ configuration of the appliance:
 <td><p>Restore the Virtual Management Database (VMDB) from a previous backup.</p></td>
 </tr>
 <tr class="even">
-<td><p><strong>Configure Database</strong></p></td>
+<td><p><strong>Configure Application</strong></p></td>
```

Comment:
Curious that this file is named as a markdown file but has HTML in it... cc @dehawkins512

Reply:
yeah, 🤷

Reply:
@Fryguy I thought this was residual from the ascii_doc conversion to markdown.